components/plan-build/bin/public.bash

```diff
@@ -139,9 +140,10 @@ trim() {
 pkg_path_for() {
   local dep="$1"
   local e
-  local cutn="$(($(echo $HAB_PKG_PATH | grep -o '/' | wc -l)+2))"
```

Comment:
Following the pattern of the rest of the local variables, should `local cutn` be on its own line here?  Also, it might be worth renaming that variable to something less prone to a bad typo.

Reply:
Yes! Good catch. I'm not sure why `shellcheck` missed that. I may file an issue with the tool.